spec/swagger/diff/diff_spec.rb

```diff
@@ -29,7 +29,8 @@
                                  'missing request param: obj/thing (type: integer)',
                                  'missing request param: str (type: string)'
                                 ],
-               'put /b/{}' => ['new required request param: extra'] },
+               'put /b/{}' => ['new required request param: extra'],
+               'post /c/' => ['new required request param: existing/b'] },
```

Comment:
There was no existing test coverage for this scenario. The problem happens when a new child is added. If that child has required params, it flags as backwards-incompatible, but it's backwards-compatible *if* the child is optional.